packages/vm/tests/StatelessRunner.js

```diff
@@ -0,0 +1,194 @@
+const ethUtil = require('ethereumjs-util')
+const BN = ethUtil.BN
+const { getRequiredForkConfigAlias, setupPreConditions, makeTx, makeBlockFromEnv } = require('./util')
+const Account = require('@ethereumjs/account').default
+const Trie = require('merkle-patricia-tree').SecureTrie
+const { default: Common } = require('@ethereumjs/common')
+const { default: VM } = require('../dist/index.js')
+const { default: DefaultStateManager } = require('../dist/state/stateManager')
+
+async function runTestCase (options, testData, t) {
+  let expectedPostStateRoot = testData.postStateRoot
+  if (expectedPostStateRoot.substr(0, 2) === '0x') {
+    expectedPostStateRoot = expectedPostStateRoot.substr(2)
+  }
+
+  // Prepare tx and block
+  let tx = makeTx(testData.transaction)
+  let block = makeBlockFromEnv(testData.env)
+  tx._homestead = true
+  tx.enableHomestead = true
+  block.isHomestead = function () {
+    return true
+  }
```

Comment:
```suggestion
```

I don't think these homstead lines are needed, as I've been removing them in other refactors/PRs without issue.